job-roles/Technical-writer.md

```diff
@@ -0,0 +1,22 @@
+# Tecnical Writer
+At MayaData we believe that the best way to deliver storage and related services to containerized and cloud-
+native environments is with containerized and cloud-native architectures. Everything we do is focused on providing data agility and simplifying the daily operation of Kubernetes SRE’s and DevOps teams using 
+Kubernetes as a data layer. We enable our users to handle complex data management operations easily, and we have become open source leaders in building Kubernetes into a first-class data platform.
+
+## Requirements:
+* MUST have grown up in a primarily English speaking location, or recently have spent 2 years in an English-speaking location doing full immersion learning
+* MUST be curious about people and technology and how people use technology to get things done
+* SHOULD have an interest in open source or software development
+* SHOULD have some interest in technical marketing or technical communication
+* SHOULD have some understanding of SEO and related concepts
+* MUST have a sample of work (at least two articles or pieces of content) that they can share as part of a portfolio
+* MUST be willing to spend 30 minutes discussing the work with an interviewer
+* MUST be willing to learn about a technical subject that they’re unfamiliar with, and explain the subject back to their teacher
+* MUST be comfortable with morning meetings, in order to accommodate time zone differences
+* SHOULD be starting their career, or looking for a switch in careers
+* MUST have internet access, and be comfortable working at a remote-first company
```

Comment:
Please, change language to be closer to other job descriptions. We don't write RFC here and would not like to scare off potential candidates.